src/appengine/handlers/cron/fuzzer_and_job_weights.py

```diff
@@ -20,19 +20,32 @@
 from base import utils
 from datastore import data_handler
 from datastore import data_types
+from datastore import ndb
 from datastore import ndb_utils
 from google_cloud_utils import big_query
 from handlers import base_handler
 from libs import handler
 from metrics import fuzzer_stats
 from metrics import logs
+from system import environment
 
 QuerySpecification = collections.namedtuple(
     'QuerySpecification', ['query_format', 'formatter', 'reason'])
 
 SpecificationMatch = collections.namedtuple('SpecificationMatch',
                                             ['new_weight', 'reason'])
 
+DEFAULT_MULTIPLIER = 10.0  # Used for blackbox and unrun jobs.
+DEFAULT_SANITIZER_WEIGHT = 0.1
+
+SANITIZER_WEIGHTS = {
```

Comment:
How are different architectures going to be handled?
In internal CF we don't give x86/arm* simulator equal weight to x64. If we continue assigning manual weights, will this still hold.

Is this also true for non-blackbox vs blackbox jobs?

Reply:
We currently don't give architectures special consideration. I can add a TODO here.

What do you mean re non-blackbox vs blackbox jobs? Currently we assign a default of 10 to those, but I expect that we need to set most of them manually.